### ISSUE TITLE
Buttons below resource summary are skipped by NVDA in browse mode

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/pinecone#249]
**Describe the bug**
The Share button has a `aria-hidden=true` property which prevents it from being accessible to a screen reader.

**To Reproduce**
1. Start a screen reader like NVDA.
2. Use the screen reader and navigate / read through the page (not tabbing through)
3. Notice that the screen reader will read back the name of the button (i.e. "button collapsed Share") but can not focus and activate it.

**Expected behavior**
The share button should be accessible and operable through a screen reader. Perhaps implemented similar to the Language menu.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox 73, NVDA

**Comment by jhung** [comment]
I did some further testing:
* This issue occurs in the Resource layout, but not in the Menu molecule.
* If navigating using other means such as Button Navigation (i.e. navigate by moving button to button) or by character, all skipped buttons are focusable.

I made this Codepen using different ways of nesting buttons, including replicating the layout of the resource buttons and the menu molecule.

https://codepen.io/jhung/pen/xxGZeOz

The expectation is that:
1. the Menu layout (Example 5) browse mode should focus each button in turn as this is the behaviour seen in Pinecone.
2. the Nested Div (Example 1) should just focus the first button, and not focus any descendent buttons in browse mode.

The tested behaviour in Example 5 is different - none of the buttons are focusable in Browse mode.

The tested behaviour in Example 1 is also different - the first button is focusable, as is the second button.

This seems to me a quirk with NVDA's browse mode and may not be worth investing a lot of time to fix the behaviour if the buttons are still accessible using the screen reader's other functions.
